manifests/database/postgresql.pp

```diff
@@ -41,21 +39,24 @@
   }
 
   if $init_db {
+    # Temporary direct use of liquibase to initially migrate the candlepin database
+    # until support is added in cpdb - https://bugzilla.redhat.com/show_bug.cgi?id=1044574
```

Comment:
Given this bug is closed per 0.9 and katello 3.3 includes 3.3, can we go back to cpdb?